environments/v2f-prod/terraform/bucket.tf

```diff
@@ -0,0 +1,11 @@
+provider google-beta {
+  project = "broad-dsp-monster-prod"
+  region = "us-central1"
+  alias = "v2f"
+}
+
+resource google_storage_bucket v2f_results_bucket {
```

Comment:
Do we need to do anything with respect to permissions?

Reply:
Not yet. Eventually we'll need to make it public, so we'll add an IAM resource then.